### PR TITLE
Add SSO instruction

### LINKED ISSUE #543: Instructions how to configure SSO (OIDC/SAML)
### Is your proposal related to a problem?

Please provide documentation on how to configure UI for Apache Kafka with any SSO provider (Google/GitHub/etc)

### Describe the solution you'd like

Any document in repository with this instruction is appreciate 

### Describe alternatives you've considered

No alternatives =)
### Additional context

No context =)

### PR DESCRIPTION
**Pull request by RustamGimadiev**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing applications:)

**What changes did you make?** (Give an overview)
Add a new documentation page with instructions about SSO configuration
**Is there anything you'd like reviewers to focus on?**
will close #543

**How Has This Been Tested?** (put an "X" next to an item)

- [X] No need to
- [ ] Manually(please, describe, when necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "X" next to an item, otherwise PR will fail)
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [X] My changes generate no new warnings(e.g. Sonar is happy)
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

### CODE REVIEW

File: guides/SSO.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+After that need to provide callback URLs, in our case we will use `https://127.0.0.1:8080/login/oauth2/code/auth0`
+![Configuration](https://github.com/provectus/kafka-ui/raw/images/images/sso-configuration.png)
+This is a main parameters required for enabling SSO
+![Parameters](https://github.com/provectus/kafka-ui/raw/images/images/sso-parameters.png)
+#### Step 3
+To launch UI for Apache Kafka with enabled TLS and SSO run following:
+``` bash
+docker run -p 8080:8080 -v `pwd`/cert:/opt/cert -e AUTH_ENABLED=true -e SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_AUTH0_CLIENTID=uhvaPKIHU4ZF8Ne4B6PGvF0hWW6OcUSB -e SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_AUTH0_CLIENTSECRET=YXfRXbitY1F5GajmodifiedTujnkVr7zuW9ECCAK4TcnCio-i -e SECURITY_BASIC_ENABLED=true -e SPRING_SECURITY_OAUTH2_CLIENT_PROVIDER_AUTH0_ISSUER_URI=https://dev-a63ggcut.auth0.com/ -e SERVER_SSL_KEY_STORE_TYPE=PKCS12 -e SERVER_SSL_KEY_STORE=/opt/cert/ui-for-apache-kafka.p12 -e SERVER_SSL_KEY_STORE_PASSWORD=123456 -e SERVER_SSL_KEY_ALIAS=ui-for-apache-kafka -e SERVER_SSL_ENABLED=true -e TRUST_STORE=/opt/cert/ui-for-apache-kafka.p12 -e TRUST_STORE_PASSWORD=123456 provectuslabs/kafka-ui:0.1.0
```
Comment (germanosin) [inline guides/SSO.md:19]: Let's split this line into multiple

File: guides/SSO.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# How to configure SSO
+SSO require additionaly to configure TLS for application, in that example we will use self-signed cert, in case of use legal certificates please skip step 1.
+#### Step 1
```
Comment (germanosin) [inline guides/SSO.md:3]: Should we name it OAuth0 example? Because for others there is no need to pass certs

**Review by germanosin — CHANGES_REQUESTED** [review]

File: guides/SSO.md (hunk 1 above)
Comment (RustamGimadiev) [reply guides/SSO.md]: done, looks better

File: guides/SSO.md (hunk 2 above)
Comment (RustamGimadiev) [reply guides/SSO.md:3]: I added a command line example without a self-signed certificate and SSL configuration, so we can leave it as-is

**Pull request merged**